analyzer/src/main/kotlin/managers/Maven.kt

```diff
@@ -180,12 +178,33 @@ class Maven(name: String, analyzerConfig: AnalyzerConfiguration, repoConfig: Rep
                 }
             }.toSortedSet()
 
-            return pkg.toReference(dependencies = dependencies)
+            val localProjects = localProjectBuildingResults.mapValues { it.value.project }
+
+            return if (localProjects.contains(identifier)) {
+                val id = Identifier(
+                        type = "Maven",
+                        namespace = node.artifact.groupId,
+                        name = node.artifact.artifactId,
+                        version = node.artifact.version
+                )
+
+                log.info { "'${id.toCoordinates()}' refers to a local project." }
```

Comment:
I did not check what is logged before this message, but it might miss some context. Maybe prefix with "Dependency '${id.toCoordinate()}'...".

Reply:
Done.